Dockerfile.assisted-service-build

```diff
@@ -1,19 +1,23 @@
 FROM quay.io/edge-infrastructure/postgresql-12-centos7:latest
 
+COPY --from=quay.io/goswagger/swagger:v0.28.0 /usr/bin/swagger /usr/bin/goswagger
+COPY --from=quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18 /opt/swagger-codegen-cli /opt/swagger-codegen-cli
+
 ENV GOPATH=/go
 ENV GOROOT=/usr/local/go
 ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 
 USER 0
 
+RUN mkdir build && chmod g+xw -R build/
+
 RUN yum install -y epel-release \
     && yum install -y --setopt=skip_missing_names_on_install=False \
-        gcc genisoimage jq git libvirt-client libvirt-devel
+        gcc genisoimage jq git libvirt-client libvirt-devel java
```

Comment:
father please forgive my sins

Reply:
Oh sorry didn't realize this emoji is "confused" I thought it was "sad", I was just trying to show solidarity

Reply:
your solidarity has been accepted and appreciated :smile: